src/i18n/en/translation.en.json

```diff
@@ -439,7 +439,12 @@
     },
     "discussion": {
       "posted": "{{name}} initiated on {{- date}} - {{count}} comment",
-      "posted_plural": "{{name}} initiated on {{- date}} - {{count}} comments"
+      "posted_plural": "{{name}} initiated on {{- date}} - {{count}} comments",
+      "summary": "{{count}} Comment{{plural}} - {{people}} Person contributed",
+      "summary-plural": "{{count}} Comment{{plural}} - {{people}} People contributed",
```

Comment:
the plural should be with `_` underscore.
https://www.i18next.com/translation-function/plurals